src/components/bar.cpp

```diff
@@ -818,9 +810,13 @@ void bar::handle(const evt::button_press& evt) {
     }
   };
 
+  bool has_dblclick = false;
+  mousebtn double_btn = mousebtn_get_double(btn);
+  has_dblclick = m_dblclicks.at(double_btn) || m_action_ctxt->has_action(double_btn, pos) != tags::NO_ACTION;
```

Comment:
This will throw an exception if `mousebtn_get_double` returns `NONE` (this happens for scroll events). It may be easier to have `m_dblclicks` be a `std::set<mousebtn>` and just check if the set contains the given double click button